cves/CVE-2019-12308.yml

```diff
@@ -254,8 +277,13 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Distrusting inputs is the first step to fixing vulnerabilities where unexpected
+      input causes errors or undesired system behavior. Contributors to this code
+      could have caught this vulnerability if they approached the design knowing
+      to distrust all inputs since it would lead to discussions regarding what possible
+      malicious or unexpected url strings can be entered by an attacker.
```

Comment:
Solid description. Respect the point about possibly having had a discussion about possible vulnerabilities.